security/documents.md

```diff
@@ -0,0 +1,11 @@
+# Document Storage
+
+Documents that are uploaded to our Object Storage ([buyer](../buyer/issue/attach-document.md#step-1-upload-a-document-to-the-tradecloud-object-storage) / [supplier](../supplier/send-order-response/attach-document.md#step-1-upload-a-document-to-the-tradecloud-object-storage)) are securely stored in our cloud.
+
+We protect your documents and the users that receive your documents as follows:
+
+* Documents can only be uploaded or downloaded through an [encrypted connection](encryption.md)
+* Once uploaded, documents are immediately scanned for viruses, worms and trojans. The scanner's database is updated every 3 hours.
+* Only authenticated Tradecloud users can request a download link of a document.
+* If a Tradecloud user requests a download link, we first verify that the user is [authorized](authorization.md) to access this document.
```

Comment:
Make it 1 sentence:
Only authenticated and authorized Tradecloud users can request a download link of a document.
(link to both authentication and authorization pages)

Reply:
Good idea, updated.